src/components/FavouritesCard/FavouritesCard.test.jsx

```diff
@@ -0,0 +1,12 @@
+import FavouritesCard from "./FavouritesCard";
+import { customRender } from "../../utils/testUtils";
+
+it("should render the text on the display", () => {
+  const { container } = customRender(
+    <FavouritesCard>
+      <p>Hello there</p>
+    </FavouritesCard>
```

Comment:
The `<FavouritesCard>` isn't used in quite the same way as the `<Layout>`, you don't need a `p` tag inside it. Simply `<FavouritesCard/>` should do!